nss-day-cohort-41/tabloid-fullstack-first-amendment-dancers-andy
# Ba 3style post

# Description
Minor style changes in post elements and added authorize statements to controllers
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
- [] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
# Testing Instructions
Pull this branch down and run it.  Ensure that all previously working features still work.  Ensure that the post elements are aesthetically pleasing.
# Checklist:
